pkg/build/buildutil/util.go

```diff
@@ -275,3 +276,29 @@ func LabelValue(name string) string {
 	klog.Warningf("In creating the value of the build label in the build pod, several attempts at manipulating %s to meet k8s label name requirements failed", name)
 	return name
 }
+
+func fromMatch(from1 k8stypes.NamespacedName, from2 k8stypes.NamespacedName) bool {
+	if from1.Namespace == from2.Namespace && from1.Name == from2.Name {
```

Comment:
This is only used once, so maybe it doesn't make sense to be it's own function, but if you keep it as it's own function, it could be simplified as `return from1.Namespace == from2.Namespace && from1.Name == from2.Name`

Reply:
yeah it looked klunky inlineing this with the k8styipes.NamespacedName stuff on line 299

but sure I'll switch to the return